src/main/java/org/generationcp/middleware/operation/builder/DataSetBuilder.java

```diff
@@ -197,15 +197,15 @@ protected VariableTypeList filterDatasetVariables(final VariableTypeList variabl
 		return newVariables;
 	}
 
-	private List<Integer> getVariablesOfSiblingDatasets(final int datasetId) {
+	public List<Integer> getVariablesOfSiblingDatasets(final int datasetId) {
```

Comment:
This is not used anymore, do you know the reason why this method is used before? Have you checked other functionalities that call this method?

Reply:
getVariablesOfSiblingDatasets was refactored last year, maybe that caused the bug? 

https://github.com/IntegratedBreedingPlatform/Middleware/commit/6645558deb70e30f4752f94dd98a148fa5d53e1d#diff-ae8f7c46d0cbcc2cb851c3bd8d7f09e0L162

Reply:
Hi Aldrin, upon investigation before, this method is used to exclude variables that exists both in study's dataset. I believe its purpose is to exclude DATASET types. Comparing to Browse Study, w/c filters only by variable type. It is only used in buildCompleteDataset.isMeasurementDataset